lang/python/pymysql/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) Alexandru Ardelean <ardeleanalex@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pymysql
+PKG_VERSION:=0.9.3
+PKG_RELEASE:=1
+
+PYPI_NAME:=PyMySQL
+PKG_HASH:=d8c059dcd81dedb85a9f034d5e22dcb4442c0b201908bede99e306d65ea7c8e7
+
+PKG_MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-pymysql
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Pure Python MySQL Client
+  URL:=https://pymysql.readthedocs.io/
+  DEPENDS:=+python3 +python3-cryptography
```

Comment:
It looks like cryptography is part of the [`rsa`](https://github.com/PyMySQL/PyMySQL/blob/v0.9.3/setup.py#L21) dependencies, which the [installation guide](https://pymysql.readthedocs.io/en/latest/user/installation.html) says to install to use `sha256_password` or `caching_sha2_password` authentication. Would that make cryptography an optional dependency?

Reply:
yep; is optional;

i can make a new change to make it selectable;